app/build.gradle.kts

```diff
@@ -31,6 +31,17 @@ android {
     namespace = "com.gemwallet.android"
     compileSdk = 34
     ndkVersion = "26.1.10909125"
+    setFlavorDimensions(listOf("default"))
+
+    productFlavors {
+        create("google-notification") {
```

Comment:
should we use channel names like google play? not features, or make this default, f-droid is optional